src/main/java/az/gdg/authservice/security/bean/CustomUserDetails.java

```diff
@@ -0,0 +1,69 @@
+package az.gdg.authservice.security.bean;
+
+import az.gdg.authservice.security.role.Role;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.List;
+
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class CustomUserDetails implements UserDetails {
```

Comment:
class/ interface names should be only singular, you can read about it here under "Singular or plural nouns?" subtopic : 
http://dolszewski.com/java/java-class-naming-ultimate-guideline/
Read the whole document please, because there are also other important naming conventions that need to be followed.